package.json

```diff
@@ -17,7 +17,8 @@
     "cy:run": "cypress run",
     "cy:run:development": "cypress run --env configFile=development",
     "cy:run:production": "cypress run --env configFile=production",
-    "test:e2e": "start-server-and-test 'yarn start --no-browser' 3000 cy:run:development"
+    "test:e2e": "start-server-and-test 'yarn start --no-browser' 3000 cy:run:development",
+    "test": "react-app-rewired test"
```

Comment:
Quick question. If we know that `react-app-rewired` is going down why don't we use `jest` directly?

Reply:
because the config of webpack and babel that rewired does. we can chat more about it.